StepDefinitions/Hotels/HotelSD.js

```diff
@@ -0,0 +1,284 @@
+const { Given, Then, When } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const HotelsFeedback = require("../../Pages/Hotels/HotelsFeedback");
+const HotelsHome = require("../../Pages/Hotels/hotelsHome");
+const HotelsSignIn = require("../../Pages/Hotels/HotelsSignIn");
+const HotelsSignUp = require("../../Pages/Hotels/hotelsSignUp");
+const Moments = require("../../Pages/Moments");
+
+const hotelsHome = new HotelsHome
+const hotelsSignin = new HotelsSignIn
+const hotelsSignUp = new HotelsSignUp
+const hotelsFeedback = new HotelsFeedback
+const moment = new Moments
+
+
+
+// TC18
+
+Given(/^I am on Hotels landing page$/, async function() {
+    await browser.url('https://www.hotels.com/');
+    await browser.pause(3000);   
+})
+
+When(/^I click on Travelers$/, async function() {
+    await hotelsHome.clickTraveler();
+    await browser.pause(3000);       
+})
+
+When(/^I enter 6 Adults$/, async function() {
+    await hotelsHome.clickAdultsIncrease();
+    await hotelsHome.clickAdultsIncrease();
+    await hotelsHome.clickAdultsIncrease();
+    await hotelsHome.clickAdultsIncrease();
+    await browser.pause(3000);       
```

Comment:
pause() isn't efficient way for waiting, it'll just stop execution for few sec then it'll start execution again doesn't matter whether the required element is there or not. So we should use waitForDisplayed() or waitForEnabled(). If the element isn't displayed test gonna fail. And that is professional.